src/e2e/puppeteer/__tests__/thought-trim.ts

```diff
@@ -0,0 +1,27 @@
+import click from '../helpers/click'
+import getEditingText from '../helpers/getEditingText'
+import paste from '../helpers/paste'
+import press from '../helpers/press'
+
+vi.setConfig({ testTimeout: 20000, hookTimeout: 20000 })
+
+it('when cursor is null, thought will be trimmed.', async () => {
+  const importText = `
+  -    Hello World.      `
+
+  await paste(importText)
+  await click('#content')
+  const editingText = await getEditingText()
+  expect(editingText).toBe(`Hello World.`)
+})
+
+it('when change the row, thought will be trimmed.', async () => {
+  const importText = `
+  -       Hello World.      `
+
+  await paste(importText)
```

Comment:
It would be good to have one test for paste and one test that creates a thought and types a value with spaces at the beginning/end. Otherwise we're only testing the paste case.